InnerEye/ML/Histopathology/preprocessing/create_tiles_dataset.py

```diff
@@ -217,11 +291,15 @@ def main(panda_dir: Union[str, Path], root_output_dir: Union[str, Path], level:
 
 
 if __name__ == '__main__':
-    main(panda_dir="/tmp/datasets/PANDA",
-         root_output_dir="/datadrive",
-         level=1,
+    from InnerEye.ML.Histopathology.datasets.tcga_prad_dataset import TcgaPradDataset
```

Comment:
If TcgaPrad is removed also this block should be removed - is it a problem we don't actually have a single dataset implementation that is compatible with this script

Reply:
Following your separate suggestion, I've decided to keep TCGA-PRAD as an example, and added a clarifying comment here.